lib/src/base-styles/_nx-page.scss

```diff
@@ -11,8 +11,10 @@
   display: grid;
   grid-template-columns: auto 1fr auto;
```

Comment:
A really long main title can completely squeeze out the subtitle.  Do we want to guard against that in any way (like a min-width on the subtitle)?
![image](https://user-images.githubusercontent.com/3630091/115409678-f9ff0f80-a1bf-11eb-8550-34bcb0202b26.png)

Reply:
@dturner32 ^^

Reply:
> A really long main title can completely squeeze out the subtitle. Do we want to guard against that in any way (like a min-width on the subtitle)?
> ![image](https://user-images.githubusercontent.com/3630091/115409678-f9ff0f80-a1bf-11eb-8550-34bcb0202b26.png)

I'll have to mock something up as an example, but yes I think the subtitle should have a min width in a case like this one

Reply:
> A really long main title can completely squeeze out the subtitle. Do we want to guard against that in any way (like a min-width on the subtitle)?
> ![image](https://user-images.githubusercontent.com/3630091/115409678-f9ff0f80-a1bf-11eb-8550-34bcb0202b26.png)

the min width for the subtitle is 150px 
![Screen Shot 2021-04-21 at 3 48 15 PM](https://user-images.githubusercontent.com/62309246/115612995-df10c600-a2b9-11eb-8eff-9817efc95c0b.png)

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/6133fe086af3b375fd557cd5741d286c92489469